RELEASING.adoc

```diff
@@ -1,35 +1,57 @@
 = Spring Cloud GCP release process
 
-The actual release process is triggered by the release admin and happens together with the rest of the Spring Cloud release.
-In order to simplify that process, here is a list of prerequisites to be performed before asking the release admin to run the release scripts.
+The actual release process is triggered by the release admin.
+As opposed to the 1.x releases, the 2.0 release happens independent of the Spring Cloud release train.
+To simplify the process, perform this list of prerequisites before asking the release admin to run the release scripts.
 
-. Ensure that modules that shouldn't be released have the `maven-deploy-plugin` configuration set to `<skip>true</skip>`.
+. Ensure that modules that shouldn't be released are not included in the `<modules>` block in the `release` profile of `pom.xml`.
 For GA releases this should be configured under the `central` Maven profile.
 +
 [source,xml]
 ----
-<build>
-    <plugins>
-        <plugin>
-            <artifactId>maven-deploy-plugin</artifactId>
-            <configuration>
-                <skip>true</skip>
-            </configuration>
-        </plugin>
-    </plugins>
-</build>
+<profile>
+    <id>release</id>
+    <modules>
+        <!-- POM Modules-->
+        <module>spring-cloud-gcp-starters</module>
+        <module>spring-cloud-gcp-dependencies</module>
+
+        <!-- Code Modules-->
+        <module>spring-cloud-gcp-autoconfigure</module>
+        <module>spring-cloud-gcp-bigquery</module>
+        <module>spring-cloud-gcp-cloudfoundry</module>
+        <module>spring-cloud-gcp-core</module>
+        <module>spring-cloud-gcp-data-datastore</module>
+        <module>spring-cloud-gcp-data-firestore</module>
+        <module>spring-cloud-gcp-data-spanner</module>
+        <module>spring-cloud-gcp-logging</module>
+        <module>spring-cloud-gcp-pubsub</module>
+        <module>spring-cloud-gcp-pubsub-stream-binder</module>
+        <module>spring-cloud-gcp-security-iap</module>
+        <module>spring-cloud-gcp-storage</module>
+        <module>spring-cloud-gcp-secretmanager</module>
+        <module>spring-cloud-gcp-security-firebase</module>
+        <module>spring-cloud-gcp-vision</module>
+        ...
+    </modules>
+    ...
+</profile>
 ----
 
-. Wait for the release scripts to have been run by the release admin.
+. Ensure all Maven commands listed in `.kokoro/stage.sh` script can be executed successfully locally.
+You can add `-Dgpg.skip` in place of the actual GPG key locally.
+
+. Invoke the Kokoro `stage` job.
```

Comment:
Can you list the full name of the job like we did [here](https://github.com/GoogleCloudPlatform/cloud-spanner-r2dbc/blob/master/RELEASING.adoc)? This will help whoever is doing it to find the job.